ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
+- Participate in another reward-bearing AF program
+- Commit the rewards to an xGov DAO for a period
+ 
+The foundation will maintain lists of eligible reward-bearing programs, and a list of qualified DAOs.
+ 
+#### Example
+Alice participates in governance during Q1 2023
+the potential reward is 5 Algo, Alice:
+1) Commit 100 Algo in an AF program
+2) Express her will to participate in xGov
+3) Will not receive 5 Algo at the end of Q1 2023. They are instead locked in one big xGov pool for 1 year
+4) Will receive 5 Algo at Q1 2024
+ 
+## Rationale
+In this ARC we seek community input on issues such as, but not limited to:
+- The types of reward-bearing activities that **SHOULD** be included in the qualification list (ie. general governance)
+- Whether xGovs **MUST** vote on all measures to keep their status
```

Comment:
It's a definite no from me. Adding a minimum participation threshold to votes makes a lot more sense to me. Different xGovs have different interests but they will likely have a community behind them that they can convince to vote if it's important for them. Forcing NFT people to vote on DEX votes will just create noise as it feel like there is little overlap between the two communities atm.

Reply:
You can always make this a must, but also leave an opt-on option on every vote if you want to track activity but not force someone to answer.

Reply:
We need more than a binary Yes/No vote. A few more options like Abstain and Veto could be useful.

Reply:
After some discussion, we tend to have a threshold/number of votes ( still to be determined and depends on [ARC-34](https://github.com/algorandfoundation/ARCs/pull/151))

We think having 3 choices (Yes/No/Neutral) might be enough.

Reply:
We really need to look into Conviction voting or Continuous approval voting.